awscli/customizations/s3/subcommands.py

```diff
@@ -392,6 +392,15 @@
 }
 
 
+FORCE_GLACIER_TRANSFER = {
```

Comment:
Do we want to name it ``--force-glacier-transfer`` or ``--force-glacier``? I think I would prefer ``--force-glacier`` because it is shorter. Also in your functional test, you use ``--force-glacier``. What do you think?